Added simpler options to GamepadManager

LeozinDaProgramacao opened the pull request:
Added an option on GamepadManager to just get the basic gamepad from the SDK using a simple funtion. Also added a subsystem example that uses this feature to show how one could do that.

LeozinDaProgramacao committed:
added simple options to GamepadManager

Added an option on GamepadManager to just get the basic gamepad from the SDK using a simple funtion. Also added a subsystem example that does not use the GamepadManager to show how one could do that.

pedrohenriquelacerda commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/gamepad/GamepadManager.java:
```diff
@@ -28,11 +43,32 @@ public static GamepadManager use(@NonNull Gamepad driver) {
+     *
+     * @return Gamepad assingned to the Driver
+     */
+    public Gamepad getDriverBasic() {
```
Maybe should change name to "getRawGamepad1" or "getRawDriverGamepad" or other name with same meaning for better understanding

LeozinDaProgramacao replied:
Alright. Implemented this fix today. 👍

pedrohenriquelacerda commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/core/lib/gamepad/GamepadManager.java:
```diff
@@ -28,11 +43,32 @@ public static GamepadManager use(@NonNull Gamepad driver) {
+     *
+     * @return Gamepad assingned to the operator
+     */
+    public Gamepad getOperatorBasic() {
```
Maybe should change name to "getRawGamepad2" or "getRawOperatorGamepad" or other name with same meaning for better understanding

LeozinDaProgramacao committed:
Implemented suggested fixes

Changed the name from "get(Operator/Driver)basic()" to a more understandable manner "getRaw(Operator/Driver)Gamepad()"
